bvm/ballerina-config/src/main/java/org/ballerinalang/config/ConfigRegistry.java

```diff
@@ -330,6 +331,34 @@ public double getAsFloat(String sectionHeader, String field) {
         return getAsMap(getConfigKey(sectionHeader, field));
     }
 
+    /**
+     * Retrieves the config value specified by the key as a List.
+     *
+     * @param key The key of the config entry
+     * @return The associated config value
+     */
+    public List getAsArray(String key) {
+        if (key == null) {
+            return null;
+        }
+        Object value = configEntries.get(key);
+        if (value instanceof List) {
+            return (List) value;
+        }
+        throw new IllegalArgumentException("config key '" + key + "' does not map to a valid 'array'");
```

Comment:
Shall we refrain from placing "array" within quotes? The other types were quoted because those are actual types + keywords.

Reply:
Changed code to return an empty List instead of throwing the exception.